applications/debug/unit_tests/nfc/nfc_test.c

```diff
@@ -17,6 +19,13 @@
 #define NFC_TEST_SIGNAL_SHORT_FILE "nfc_nfca_signal_short.nfc"
 #define NFC_TEST_SIGNAL_LONG_FILE "nfc_nfca_signal_long.nfc"
 #define NFC_TEST_DICT_PATH EXT_PATH("unit_tests/mf_classic_dict.nfc")
+#define NFC_TEST_NFCA_PATH EXT_PATH("nfc/nfca.nfc")
+#define NFC_TEST_MFC_1K_4B_PATH EXT_PATH("nfc/mf_classic_1k_4b.nfc")
+#define NFC_TEST_MFC_4K_4B_PATH EXT_PATH("nfc/mf_classic_4k_4b.nfc")
+#define NFC_TEST_MFC_1K_7B_PATH EXT_PATH("nfc/mf_classic_1k_7b.nfc")
+#define NFC_TEST_MFC_4K_7B_PATH EXT_PATH("nfc/mf_classic_4k_7b.nfc")
+#define NFC_TEST_MFU_PATH EXT_PATH("nfc/mf_ultralight.nfc")
+#define NFC_TEST_MFDES_PATH EXT_PATH("nfc/mf_desfire.nfc")
```

Comment:
Use the unit_tests folder for resources needed for unit testing. Also add the unit test assets to the assets folder.